parser/src/main/java/com/distkv/parser/DistkvNewSQL.g4

```diff
@@ -92,6 +92,14 @@ intIncrDefault: key;
 // Increase delta points
 intIncrDelta: key anyInt;
 
+// expire
+expireStatement: expireStr | expireList | expireDict | expireSet | expireSlist | expireInt;
+expireStr: 'expire.str' key anyInt;
```

Comment:
how about just using `expire` key anyInt; because the expire operation is common enough. how do you think.

Reply:
The existing data structure does not support this implementation, we will resolve it later.